Library/Homebrew/dev-cmd/pr-pull.rb

```diff
@@ -24,6 +24,9 @@ def pr_pull_args
              description: "Download the bottles but don't upload them."
       switch "--no-commit",
              description: "Do not generate a new commit before uploading."
+      switch "--no-long-build-check",
```

Comment:
When would you want to pass this?

Reply:
When the automerge in homebrew-core fails due to the conflict check: you need to manually pr-pull with that flag to overcome the automerge failure. Of course you need to review the conflict manually:
- is the corresponding long build PR currently running / being worked on
- is the corresponding long build PR stale? If yes, maybe add a comment there that you merged something and a rebase will be needed.

Reply:
It feels like a nicer solution here might be to remove the label instead?

Reply:
Can we ship without this flag for now and see if we need it? My concern is that it's so destructive if someone passes it and not visible to anyone else.

Reply:
Done. I removed the flag.